ASP-NET/ASP-NET/Classes/Options.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ASP_NET.Classes;
+
+public class Options : IOptions
+{
+    public string JwtKey { get; } = string.Empty;
+    public string JwtIssuer { get; } = string.Empty;
+    public string JwtAudience { get; } = string.Empty;
+
+    public Options(IConfiguration configuration)
+    {
+        this.JwtKey = configuration["Jwt:Key"];
+        this.JwtIssuer = configuration["Jwt:Issuer"];
+        this.JwtAudience = configuration["Jwt:Audience"];
+    }
+}
+
+public interface IOptions
```

Comment:
You dont need this class, you just need JwtOptions, and you need to configure it like this https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0